addons/web_editor/static/src/js/editor/transcoder.js

```diff
@@ -242,6 +280,11 @@ function classToStyle($editable) {
  * @param {jQuery} $editable
  */
 function styleToClass($editable) {
+    // Outlook revert
+    $editable.find('table.o_outlook_hack').each(function () {
+        $(this).after($('a', this));
```

Comment:
Wut ? I do not understand that line. And don't you want to just extract the button from the table ?